File: sample/src/main/java/com/mikepenz/aboutlibraries/sample/ExtendActivity.java
package com.mikepenz.aboutlibraries.sample;

import android.os.Bundle;

import com.mikepenz.aboutlibraries.Libs;
import com.mikepenz.aboutlibraries.ui.LibsActivity;


public class ExtendActivity extends LibsActivity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        /*
        Intent intent = new Intent();
        intent.putExtra(Libs.BUNDLE_FIELDS, Libs.toStringArray(R.string.class.getFields()));
        intent.putExtra(Libs.BUNDLE_LIBS, new String[]{"activeandroid", "caldroid"});
        setIntent(intent);
        */

        setIntent(new Libs.Builder().withFields(R.string.class.getFields()).withLibraries("activeandroid", "caldroid").intent(this));


        super.onCreate(savedInstanceState);
    }
}

File: app/src/androidMain/kotlin/com/mikepenz/aboutlibraries/sample/ExtendActivity.kt
@file:Suppress("DEPRECATION")

package com.mikepenz.aboutlibraries.sample

import android.os.Bundle
import com.mikepenz.aboutlibraries.LibsBuilder
import com.mikepenz.aboutlibraries.ui.LibsActivity


class ExtendActivity : LibsActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        /*
        val intent = Intent()
        intent.putExtra(Libs.BUNDLE_FIELDS, Libs.toStringArray(R.string::class.java.fields))
        intent.putExtra(Libs.BUNDLE_LIBS, arrayOf("activeandroid", "caldroid"))
        setIntent(intent)
        */
        intent = LibsBuilder()
            .withEdgeToEdge(true)
            .withSearchEnabled(true)
            .intent(this)
        super.onCreate(savedInstanceState)
    }
}
